src/dotnet-format.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFrameworks>netcoreapp2.1;netcoreapp2.2;netcoreapp3.0</TargetFrameworks>
```

Comment:
we already specify that this tool can run on newer runtimes [here](https://github.com/dotnet/format/blob/master/src/runtimeconfig.template.json).  nuget not restoring it correcty in the 3.0 SDK is a bug that I am told is being fixed with the Local Tools SDK feature

Reply:
> nuget not restoring it correcty in the 3.0 SDK is a bug

It should not be able to directly restored by nuget (as in adding to csproj and run restore). It is designed that way. You could use "tool restore" with local tools.